fixtures/chauvet-professional/colorado-1-solo.json

```diff
@@ -87,47 +82,63 @@
         {
           "dmxRange": [11, 30],
           "type": "ColorPreset",
-          "comment": "R: 100% G: 0-100% B: 0 "
+          "comment": "Red ... Yellow",
```

Comment:
Please use `…` (one character) instead of `...` (three characters) here and in the following capabilities.